random_numbers/random_numbers.F90

```diff
@@ -86,7 +86,7 @@ subroutine getRandomNumber_Scalar(stream, number)
     type(randomNumberStream), intent(inout) :: stream
     real,                     intent(  out) :: number
 
-    number = getRandomReal(stream%theNumbers)
+    number = real(getRandomReal(stream%theNumbers))
```

Comment:
getRandomReal is already coming back as a real(kind=8) value.  Is this to continue to allow r4 compilation?

Reply:
Yeah that'll just cast to default so it'll always be the same kind